hazelcast/src/main/java/com/hazelcast/config/AbstractXmlConfigBuilder.java

```diff
@@ -18,6 +18,10 @@
 
```

Comment:
We should move this to `internal`. `AbstractYamlConfigBuilder` is already there.

Reply:
Thing is, this class is inherited by public API classes -`XmlConfigBuilder` and others. So it should be obvious to any employee in HZ that they are changing methods on a public class. Because of that, `AbstractYamlConfigBuilder` should probably be moved back. WDYT?

Reply:
Good thinking :+1: Let's move `AbstractYamlConfigBuilder` back then.